man/systemd.service.xml

```diff
@@ -63,6 +63,22 @@
     <para>The <citerefentry><refentrytitle>systemd-run</refentrytitle><manvolnum>1</manvolnum></citerefentry>
     command allows creating <filename>.service</filename> and <filename>.scope</filename> units dynamically
     and transiently from the command line.</para>
+
+    <para>In addition to the various drop-in behaviors described in
+    <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+    services also support a top-level drop-in with <filename>-.service.d/</filename> that allows
+    modifying the settings of all services on the system. Drop-in files must contain appropriate section headers.
```

Comment:
Hmm, maybe "adding" instead of "modifying" for this whole section, since it can't really do much except add?

Reply:
I also think it's fine to just have a note to defer to `systemd.unit` instead of mentioning the section headers.

Reply:
> Hmm, maybe "adding" instead of "modifying" for this whole section, since it can't really do much except add?

Some settings you can "delete" or "reset" with the right syntax, e.g. `CapabilityBoundingSet=~`. How about "alter or add" since that's the language used in `systemd.unit`.

Reply:
Huh, but if it's in `-.service.d`, it's too low in the chain to do that stuff though, right? Not a biggie though.